package.json

```diff
@@ -25,6 +27,8 @@
   },
   "devDependencies": {
     "bl": ">=0.9.0 <0.10.0-0",
+    "coveralls": "^2.11.2",
```

Comment:
@rvagg any reason?  Just curious (happy if you just throw a blog link my way),  wouldn't `^2.11.0` catch more bugs?